pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -458,7 +458,7 @@ func (p *PriorityQueue) Update(oldPod, newPod *v1.Pod) error {
 	// If pod is not in any of the queues, we put it in the active queue.
 	err := p.activeQ.Add(p.newQueuedPodInfo(newPod))
 	if err == nil {
-		p.PodNominator.AddNominatedPod(newPod, "")
+		p.PodNominator.AddNominatedPod(framework.NewPodInfo(newPod), "")
```

Comment:
lets do this here so we avoid creating podInfo twice:
```
pInfo := p.newQueuedPodInfo(newPod)
if err := p.activeQ.Add(pInfo); err != nil {
 return err
}
p.PodNominator.AddNominatedPod(pInfo, "")
p.cond.Broadcast()
return nil
```

Reply:
done.